src/main/kotlin/org/jetbrains/research/ml/codetracker/tracking/TaskFileHandler.kt

```diff
@@ -76,9 +74,11 @@ object TaskFileHandler {
             logger.info("Project $project is already added or set to be added")
             return
         }
-        SurveyUiData.programmingLanguage.currentValue?.let {
-            initProject(project)
-        } ?: projectsToInit.add(project)
+        if (MainController.visiblePane != SurveyControllerManager) {
+            SurveyUiData.programmingLanguage.currentValue?.let {
+                initProject(project)
+            } ?: projectsToInit.add(project)
+        }
```

Comment:
Don't forget to add it to the `projectsToInit` if the visible pane is not a SurveyControllerManager